src/components/graphs/GraphContainer.js

```diff
@@ -147,54 +147,53 @@ const GraphContainer = () => {
         />
       </header>
 
-      <section id="lineGraph" className="graph-container">
-        {filtered.length > 0 ? (
-          <div className="all-graphs">
-            <h2 style={{ marginTop: '2rem' }}>
-              Incident reports identified by our data collection methods per
-              month
-            </h2>
-            <p>
-              <h4>April 2020 - Present</h4>
-            </p>
-          </div>
-        ) : null}
-
-        {filtered.length > 0 ? (
-          <LineGraph data={counts} months={months} />
-        ) : (
-          noDataDisplay()
-        )}
-      </section>
-      {filtered.length > 0 ? (
-        <div>
-          <section id="barGraph" className="graph-container">
-            <div>
-              <h2 style={{ marginTop: '5rem' }}>
-                Total incident reports identified by our data collection methods
-                by state
+      <div className="all-graphs-container">
+        <section id="lineGraph" className="graph-container">
+          {filtered.length > 0 ? (
+            <div className="all-graphs">
+              <h2 style={{ marginTop: '2rem' }}>
+                Incident reports identified by our data collection methods per month
               </h2>
+              <p>
+                <h4>April 2020 - Present</h4>
+              </p>
             </div>
-            <BarGraph count={barCounts} />
-          </section>
-
-          <section id="pieGraph" className="graph-container">
-            <h2 style={{ marginTop: '5rem' }}>
-              Prevalence of Force Ranks as identified by our data collection
-              methods
-            </h2>
-
-            <div className="pie-holder">
-              <div className="pie">
-                <PieGraph data={filtered} />
+          ) : null}
+
+          {filtered.length > 0 ? (
+            <LineGraph data={counts} months={months} />
+          ) : (
+            noDataDisplay()
+          )}
+        </section>
+        {filtered.length > 0 ? (
+          <div>
+            <section id="barGraph" className="graph-container">
+              <div>
+                <h2 style={{ marginTop: '5rem' }}>
+                  Total incident reports identified by our data collection methods by
+                  state
+                </h2>
               </div>
-              <div className="pie-legend">
-                <Legend />
+              <BarGraph count={barCounts} />
+            </section>
+            <section id="pieGraph" className="graph-container">
+              <h2 style={{ marginTop: '5rem' }}>
+                Prevalence of Force Ranks as identified by our data collection
+                methods
+              </h2>
+              <div className="pie-holder">
+                <div className="pie">
+                  <PieGraph data={filtered} />
+                </div>
+                <div className="pie-legend">
```

Comment:
I saw the way it was shown on your local machine, this looks better than the current iteration of the map page.